modules/FormModel.js

```diff
@@ -581,69 +907,28 @@
     },
 
     /**
-     * @param [options] {Object} See initialize option's 'model', 'fields', 'models', 'computed'.
-     * @private
+     * Sets the mapping using the form model's default mapping or the options mappings if available.
+     * Also sets the tracked models if options.models is provided.
      * @method __initMappings
-     */
-    __initMappings: function(options) {
-      var defaultMapping = _.result(this, 'mapping'),
-        optionsMapping = _.pick(options, ['model', 'fields', 'models', 'computed']);
-      this.__initModels(optionsMapping, defaultMapping);
-      this.__initComputeds(optionsMapping, defaultMapping);
-    },
-
-    /**
-     * @param [optionsMapping] {Object} a mapping object with override values
-     * @param [defaultMapping] {Object} the default mapping object
+     * @param [options] {Object} See initialize options: 'mapping' and 'models'.
      * @private
-     * @method __initModels
      */
-    __initModels: function(optionsMapping, defaultMapping) {
-      var modelConfigs = this.__pullModelsFromMapping(optionsMapping) || this.__pullModelsFromMapping(defaultMapping);
-      _.each(modelConfigs, this.addModel, this);
-    },
-
-    /**
-     * @param [optionsMapping] {Object} a mapping object with override values
-     *   @param [optionsMapping.computed] {Array} an array of Computed Configs
-     * @param [defaultMapping] {Object} the default mapping object
-     *   @param [defaultMapping.computed] {Array} and array of Computed Configs
-     * @private
-     * @method __initComputeds
-     */
-    __initComputeds: function(optionsMapping, defaultMapping) {
-      var computeds;
-      optionsMapping = optionsMapping || {};
-      defaultMapping = defaultMapping || {};
-      computeds = optionsMapping.computed || defaultMapping.computed;
-      _.each(computeds, this.addComputed, this);
-    },
-
-    /**
-     * @param [mapping] {Object} object with attributes that contain either a model/field pair as a convenience or an array of
-     *   model configs. The model/field pair takes priority if both exist.
-     * @return {Array} an array of model configs that are either from the mapping.model or mapping.model. If no model configs are
-     *   defined in the mapping, it will return null.
-     * @private
-     * @method __pullModelsFromMapping
-     */
-    __pullModelsFromMapping: function(mapping) {
-      var modelConfigs = [];
-      if (mapping && mapping.model) {
-        modelConfigs.push({
-          model: mapping.model,
-          fields: mapping.fields
-        });
-      } else if (mapping && mapping.models) {
-        modelConfigs = mapping.models.slice();
+    __initMappings: function(options) {
+      var mapping,
+        defaultMapping = _.result(this, 'mapping'),
+        optionsMapping = options.mapping;
+      mapping = options.mapping || defaultMapping;
```

Comment:
I thought about it, then decided it was more pain to do right now than necessary - specifically how you remove a mapping on a parent. We can add it later if needed? In the test formModelInitialization I show how you can use the parent's prototype to extend it and override values. I don't know. I'm on the fence and I don't have a lot of brain power or motiviation right now to think about making it better. If you have suggestings or feelings, let me know